gunicorn.py

```diff
@@ -1,8 +1,8 @@
-from src.data import start_update
+from src.db import start_update
 
 
 def on_starting(server):
-    start_update()
+    start_update(True, True, True)
```

Comment:
What do these three flags mean?

Reply:
The flags in order of use are for refreshing campus eateries, recalculating swipe data and refreshing Collegetown eateries. Refreshing an eatery means refreshing all of its  information, the hours for each eatery is always refreshed regardless of these flags.